src/main/java/pieces/Piece.java

```diff
@@ -0,0 +1,98 @@
+    public static final String BLACK = "black";
+    public static final char WHITE_PAWN_REPRESENTATION = 'p';
+    public static final char BLACK_PAWN_REPRESENTATION = 'P';
+    public static final char WHITE_KNIGHT_REPRESENTATION = 'n';
+    public static final char BLACK_KNIGHT_REPRESENTATION = 'N';
+    public static final char WHITE_ROOK_REPRESENTATION = 'r';
+    public static final char BLACK_ROOK_REPRESENTATION = 'R';
+    public static final char WHITE_BISHOP_REPRESENTATION = 'b';
+    public static final char BLACK_BISHOP_REPRESENTATION = 'B';
+    public static final char WHITE_QUEEN_REPRESENTATION = 'q';
+    public static final char BLACK_QUEEN_REPRESENTATION = 'Q';
+    public static final char WHITE_KING_REPRESENTATION = 'k';
+    public static final char BLACK_KING_REPRESENTATION = 'K';
+
+    private String color;
+    private char representation;
+
+    private Piece(final String color, final char representation) {
+        this.color = color;
+        this.representation = representation;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public char getRepresentation() {
+        return representation;
+    }
+
+    public static Piece createWhitePawnInstance() {
+        return new Piece(WHITE, WHITE_PAWN_REPRESENTATION);
+    }
+
+    public static Piece createBlackPawnInstance() {
+        return new Piece(BLACK, BLACK_PAWN_REPRESENTATION);
+    }
+
+    public static Piece createWhiteKnightInstance() {
+        return new Piece(WHITE, WHITE_KNIGHT_REPRESENTATION);
+    }
+
+    public static Piece createBlackKnightInstance() {
+        return new Piece(BLACK, BLACK_KNIGHT_REPRESENTATION);
+    }
+
+    public static Piece createWhiteRookInstance() {
+        return new Piece(WHITE, WHITE_ROOK_REPRESENTATION);
+    }
+
+    public static Piece createBlackRookInstance() {
+        return new Piece(BLACK, BLACK_ROOK_REPRESENTATION);
+    }
+
+    public static Piece createWhiteBishopInstance() {
+        return new Piece(WHITE, WHITE_BISHOP_REPRESENTATION);
+    }
+
+    public static Piece createBlackBishopInstance() {
+        return new Piece(BLACK, BLACK_BISHOP_REPRESENTATION);
+    }
+
+    public static Piece createWhiteQueenInstance() {
+        return new Piece(WHITE, WHITE_QUEEN_REPRESENTATION);
+    }
+
+    public static Piece createBlackQueenInstance() {
+        return new Piece(BLACK, BLACK_QUEEN_REPRESENTATION);
+    }
+
+    public static Piece createWhiteKingInstance() {
+        return new Piece(WHITE, WHITE_KING_REPRESENTATION);
+    }
+
+    public static Piece createBlackKingInstance() {
+        return new Piece(BLACK, BLACK_KING_REPRESENTATION);
+    }
+
+    public boolean isWhite() {
+        if (this.color.equals("white")) {
+            return true;
+        }
+        return false;
+    }
+
+    public boolean isBlack() {
+        if (this.color.equals("black")) {
+            return true;
+        }
+        return false;
+    }
+}
```

Comment:
```suggestion
public boolean isWhite() {
        return this.color.equals("white");
    }

    public boolean isBlack() {
        return this.color.equals("black");
    }
```
위 코드는 이렇게 단순화 할 수 있을 것 같아요.
그리고 문자열이 아닌 Enum으로 만들면 아래와 같이 더 직관적인 코드로 만들 수 있을 것 같네요.
다음 단계에서 Enum으로 변환하는 내용이 있으니 그 때 진행하셔도 될 것 같습니다

```java
public boolean isWhite() {
        return this.color == Color.WHITE;
    }
```

Reply:
훨씬 보기가 편하네요! 조언 감사합니다~!!